VATSpy.dat

```diff
@@ -17940,7 +17940,7 @@ LGMD-T|Makedonia-(TSL sector)|LGMD-T|LGMD-T
 LHCC|Budapest||LHCC
 LIBB|Brindisi||LIBB
 LIMM|Milano||LIMM
-LIPP|Padova|LIMM|LIMM
+LIPP|Padova|LIMM|LIPP
 LIPP|Padova||LIPP

```

Comment:
Now this line is no longer necessary. Remove it, then it's good to merge 😛

Reply:
@prryplatypus updated to remove the link this is no longer nescessary